homeassistant/components/signalmessenger/notify.py

```diff
@@ -0,0 +1,69 @@
+"""Signal Messenger for notify component."""
+import homeassistant.helpers.config_validation as cv
```

Comment:
Imports should be grouped in three groups: standard library, 3rd party and homeassistant.

https://www.python.org/dev/peps/pep-0008/#imports